Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalPassivePartial.cc

```diff
@@ -0,0 +1,156 @@
+///////////////////////////////////////////////////////////////////////////////
+// File: DDHGCalPassivePartial.cc
+// Description: Geometry factory class for the passive part of a partial
+//              silicon module
+// Created by Sunanda Banerjee
+///////////////////////////////////////////////////////////////////////////////
+
+#include <string>
+#include <vector>
+#include <sstream>
+
+#include "DD4hep/DetFactoryHelper.h"
+#include "DataFormats/Math/interface/angle_units.h"
+#include "DetectorDescription/DDCMS/interface/DDPlugins.h"
+#include "DetectorDescription/DDCMS/interface/DDutils.h"
+#include "Geometry/HGCalCommonData/interface/HGCalWaferMask.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#define EDM_ML_DEBUG
+using namespace angle_units::operators;
+
+struct HGCalPassivePartial {
+  HGCalPassivePartial() { throw cms::Exception("HGCalGeom") << "Wrong initialization to HGCalPassivePartial"; }
+  HGCalPassivePartial(cms::DDParsingContext& ctxt, xml_h e) {
+    cms::DDNamespace ns(ctxt, e, true);
+    cms::DDAlgoArguments args(ctxt, e);
+#ifdef EDM_ML_DEBUG
+    edm::LogVerbatim("HGCalGeom") << "DDHGCalPassivePartial: Creating an instance";
+#endif
+    std::string parentName = args.parentName();
+    std::string material = args.value<std::string>("ModuleMaterial");
+    double thick = args.value<double>("ModuleThickness");
+    double waferSize = args.value<double>("WaferSize");
+    double waferSepar = args.value<double>("SensorSeparation");
+#ifdef EDM_ML_DEBUG
+    edm::LogVerbatim("HGCalGeom") << "DDHGCalPassivePartial: Module " << parentName << " made of " << material << " T "
+                                  << thick << " Wafer 2r " << waferSize << " Half Separation " << waferSepar;
+#endif
+    std::vector<std::string> tags = args.value<std::vector<std::string>>("Tags");
+    std::vector<int> partialTypes = args.value<std::vector<int>>("PartialTypes");
+    std::vector<int> placementIndex = args.value<std::vector<int>>("{lacementIndex");
```

Comment:
```suggestion
    std::vector<int> placementIndex = args.value<std::vector<int>>("PlacementIndex");
```

Reply:
Thanks a lot Yana. I could not guess it from the error message.